core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -309,8 +309,13 @@ public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
   }
 
   @Override
+  public InputStream open(String path, OpenOptions options) throws IOException {
+    return openObject(stripPrefixIfPresent(path), options);
```

Comment:
If and when we add support for multiple containers within the same UFS, I expect more logic here. Also, I think its a cleaner abstraction if the the object store implementations did not have to parse an Alluxio URI to extract the object key. Let me know what you think?.